### DISCUSSION TITLE
bulk upload observations to observations.org

### DISCUSSION THREAD

**Discussion by pddpauw** [mcguirepr89/BirdNET-Pi#1082] [Show and tell]
for those interested, attached a manual on how to upload in bulk to observation.org or waarnemingen.be or waarneming.nl...
Please do data verification to avoid wrong uploads.

I've made an xls conversion file as well (using the BirdNet DB download, and converting to daily observations), if interested, let me know.

[manual_for_bulk_import_to_observations.pdf](https://github.com/mcguirepr89/BirdNET-Pi/files/13761448/manual_for_bulk_import_to_observations.pdf)

**Comment by KB3LZV** [comment]
Observations.org appears to be for sale. Nothing there.

**Reply by pddpauw** [reply]
correct, is observation.org.  Have adapted the typo on the post and pdf.

**Reply by alexbelgium** [reply]
Hi, have you ever thought about using their API ? We've started a discussion here https://github.com/Nachtzuster/BirdNET-Pi/discussions/130 to see if we can implement this feature
